test/mocha/25-http-edv-client.js

```diff
@@ -315,6 +391,105 @@ describe('bedrock-edv-storage HTTP API - edv-client', () => {
       should.exist(err);
       err.name.should.equal('NotFoundError');
     });
+    it('should get an EDV config', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f677';
+      const handle = 'testKey2';
+      const {httpsAgent} = brHttpsAgent;
+
+      const capabilityAgent = await CapabilityAgent.fromSecret(
+        {secret, handle});
+
+      const keystoreAgent = await helpers.createKeystore({capabilityAgent});
+      const invocationSigner = capabilityAgent.getSigner();
+
+      // corresponds to the passport authenticated user
+      const actor = actors['alpha@example.com'];
+
+      let edvClient;
+      let edvConfig;
+      let config;
+      let err;
+      try {
+        ({edvClient, edvConfig} = await helpers.createEdv(
+          {actor, capabilityAgent, keystoreAgent, urls}));
+        config = await EdvClient.findConfigs({
+          controller: edvConfig.controller, invocationSigner,
+          url: edvClient.id, httpsAgent
+        });
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      config.should.be.an('object');
+      config.id.should.be.a('string');
+      config.id.should.equal(edvClient.id);
+    });
+    it('should get an EDV', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f679';
+      const handle = 'testKey4';
+      const {httpsAgent} = brHttpsAgent;
+
+      const capabilityAgent = await CapabilityAgent.fromSecret(
+        {secret, handle});
+
+      const keystoreAgent = await helpers.createKeystore({capabilityAgent,
+        referenceId: 'test'});
+      // corresponds to the passport authenticated user
+      const actor = actors['alpha@example.com'];
+
+      let edvConfig;
+      let config;
+      let err;
+      try {
+        ({edvConfig} = await helpers.createEdv(
+          {actor, capabilityAgent, keystoreAgent, urls}));
+        config = await EdvClient.findConfigs({
+          controller: edvConfig.controller, referenceId: 'test',
+          url: 'https://localhost:18443/edvs', httpsAgent
+        });
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(config);
+      config.should.be.an('array');
+      config.should.have.length(1);
```

Comment:
can there be additional assertions that this config matches something that was put in?

Reply:
Added in a check for referenceId.